cypress-tests/cypress/e2e/PaymentUtils/Paybox.js

```diff
@@ -0,0 +1,528 @@
+// };
+
+// const singleUseMandateData = {
+//     customer_acceptance: {
+//         acceptance_type: "offline",
+//         accepted_at: "1963-05-03T04:07:52.723Z",
+//         online: {
+//             ip_address: "125.0.0.1",
+//             user_agent: "amet irure esse",
+//         },
+//     },
+//     mandate_type: {
+//         single_use: {
+//             amount: 8000,
+//             currency: "EUR",
+//         },
+//     },
+// };
+
+// const multiUseMandateData = {
+//     customer_acceptance: {
+//         acceptance_type: "offline",
+//         accepted_at: "1963-05-03T04:07:52.723Z",
+//         online: {
+//             ip_address: "125.0.0.1",
+//             user_agent: "amet irure esse",
+//         },
+//     },
+//     mandate_type: {
+//         multi_use: {
+//             amount: 8000,
+//             currency: "EUR",
+//         },
+//     },
+// };
+
+export const connectorDetails = {
+    card_pm: {
+        PaymentIntent: {
+            Request: {
+                payment_method: "card",
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "EUR",
+                customer_acceptance: null,
+                setup_future_usage: "on_session",
+            },
+            Response: {
+                status: 200,
+                body: {
+                    status: "requires_payment_method",
+                },
+            },
+        },
+        // "3DSManualCapture": {
+        //     Request: {
+        //         payment_method: "card",
+        //         payment_method_data: {
+        //             card: successfulThreeDSTestCardDetails,
+        //         },
+        //         currency: "EUR",
+        //         customer_acceptance: null,
+        //         setup_future_usage: "on_session",
+        //     },
+        //     Response: {
+        //         status: 200,
+        //         trigger_skip: true,
+        //         body: {
+        //             status: "requires_capture",
+        //         },
+        //     },
+        // },
+        // "3DSAutoCapture": {
+        //     Request: {
+        //         payment_method: "card",
+        //         payment_method_data: {
+        //             card: successfulThreeDSTestCardDetails,
+        //         },
+        //         currency: "EUR",
+        //         customer_acceptance: null,
+        //         setup_future_usage: "on_session",
+        //     },
+        //     Response: {
+        //         status: 200,
+        //         trigger_skip: true,
+
+        //         body: {
+        //             status: "requires_customer_action",
+        //         },
+        //     },
+        // },
+        No3DSManualCapture: {
+            Request: {
+                payment_method: "card",
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "EUR",
```

Comment:
`Currency` can be removed from every `config` except `PaymentIntent` because currency cannot be updated by any means. So it is useless having it passed in confirm call